tests/queries/0_stateless/02558_system_processes_elapsed.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
```

Comment:
Want to add `# Tags: long` due to l. 8?

Reply:
@rschu1ze it will not sleep 10 seconds each time, it will sleep only 1.5 and after it will be KILL'ed, so the time should be below 60 seconds.